matrix-sdk-android/src/main/res/values/strings.xml

```diff
@@ -82,6 +82,13 @@
     <string name="notice_room_third_party_registered_invite">%1$s accepted the invitation for %2$s</string>
     <string name="notice_room_third_party_registered_invite_by_you">You accepted the invitation for %1$s</string>
 
+    <string name="notice_widget_added">%1$s widget added by %2$s</string>
+    <string name="notice_widget_added_by_you">%1$s widget added by you</string>
+    <string name="notice_widget_removed">%1$s widget removed by %2$s</string>
+    <string name="notice_widget_removed_by_you">%1$s widget removed by you</string>
+    <string name="notice_widget_modified">%1$s widget modified by %2$s</string>
+    <string name="notice_widget_modified_by_you">%1$s widget modified by you</string>
```

Comment:
Thanks for the update with "by_you" feature. For those wordings, are they copied from Riot-Web? I think "bob added widget X" will be more coherent with other notice string.

Reply:
Yes that's copied from riot web, not sure we should change it...